alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -33,161 +26,46 @@ use crate::tty::{ChildEvent, EventedPty, EventedReadWrite};
 
 mod child;
 mod conpty;
+#[cfg(feature = "winpty")]
+mod switchable_backends;
```

Comment:
Please rename this to `automatic_backend`.

Reply:
That's a nice name!